### PR TITLE
fix(skhd): use service instead of plist

### PR DESCRIPTION
**Pull request by mvllow**
Applying #25 to skhd as well :)

Tested via:

```sh
brew uninstall skhd
brew untap koekeishiya/homebrew-formulae
brew install ./skhd.rb
```

Ref #26

### CODE REVIEW

File: skhd.rb [hunk 1]
```diff
@@ -23,70 +23,26 @@ def caveats; <<~EOS
     EOS
   end
 
-  plist_options :manual => "skhd"
-
   if build.with? "logging"
```
Comment (mvllow) [inline skhd.rb:26]: Is this option necessary? It's not present in yabai. Happy to remove and do logging by default if desired.

File: skhd.rb [hunk 2]
```diff
@@ -23,70 +23,26 @@ def caveats; <<~EOS
...
-          <string>Interactive</string>      
-          <key>Nice</key>
-          <integer>-20</integer>
-        </dict>
-        </plist>
-        EOS
-      end
+    service do
+      run "#{opt_bin}/skhd"
+      require_root true
```
Comment (hunterliao29) [inline skhd.rb:40]: Does skhd need root access?
```suggestion
```

File: skhd.rb [hunk 3]
```diff
@@ -23,70 +23,26 @@ def caveats; <<~EOS
...
+      environment_variables PATH: "#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin"
+      keep_alive true
+      interval 30
```
Comment (hunterliao29) [inline skhd.rb:43]: seem like this only request when run_type is set to interval
```suggestion
```

File: skhd.rb (hunk 2 above)
Comment (mvllow) [reply skhd.rb:40]: I'd love to remove the `require_root`. Let me test this locally when I get a chance and will commit if everything works :)

**Comment by mvllow** [comment]
Replaced by #30

**Pull request closed without merging**